### PR TITLE
Managing tasks tutorial

### LINKED ISSUE #2633: Cloud: Managing tasks
While conducting interviews for other features, some Meilisearch employees mentioned it could be good for the documentation to include instructions on how to manage tasks in Meilisearch Cloud.

Further research is necessary to determine how much user demand there is for this, and what content we need to include in a guide.

## Data collection
- [ ] User feedback (complaints, support tickets, social media/discord discussions)
- [ ] Interview product designer about intended workflow and potential issues

### PR DESCRIPTION
**Pull request by guimachiavelli**
Closes #2633

### CODE REVIEW

File: learn/async/paginating_tasks.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+<CodeSamples id="get_all_tasks_paginating_1" />
+
+Meilisearch will return a batch of tasks. Each batch of returned tasks is often called a "page" of tasks, and the size of that page is determined by `limit`:  
+
+```json
+{
+  "results": [
+    …
+  ],
+  "total": 50,
```
Comment (CaroFG) [inline learn/async/paginating_tasks.mdx:23]: Is `total`, the total number of tasks returned by 2 batches (or pages) of tasks? Or is it the absolute total number of tasks in the instance?

File: learn/async/working_with_tasks.mdx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  },
+  "error": null,
+  "duration": "PT0.001192S",
+  "enqueuedAt": "2022-08-04T12:28:15.159167Z",
+  "startedAt": "2022-08-04T12:28:15.161996Z",
+  "finishedAt": "2022-08-04T12:28:15.163188Z"
+}
+```
+
+If the task is still `enqueued` or `processing`, wait a few moments and query the database once again.
```
Comment (CaroFG) [inline learn/async/working_with_tasks.mdx:87]: Should we link here to the task webhook guide?

**Review by CaroFG — APPROVED** [review]

**Review by curquiza — APPROVED** [review]

File: .code-samples.meilisearch.yaml [hunk 3]
```diff
@@ -958,27 +958,15 @@ date_guide_sort_1: |-
...
+    -X GET 'http://localhost:7700/tasks?statuses=failed'
+async_guide_filter_by_statuses_2: |-
   curl \
-    -X GET 'http://localhost:7700/tasks?indexUids=movies'
-async_guide_canceled_by_1: |-
+    -X GET 'http://localhost:7700/tasks?statuses=failed,canceled'
+async_guide_multiple_filters_1: |-
   curl \
-    -X GET 'http://localhost:7700/tasks?canceledBy=9,15'
+    -X GET 'http://localhost:7700/tasks?indexUids=movies&types=documentAdditionOrUpdate,documentDeletion&statuses=processing'
```
Comment (curquiza) [inline .code-samples.meilisearch.yaml:969]: Is it expected 2 calls in 1 example (I'm not against, just checking)

File: learn/async/paginating_tasks.mdx (hunk 1 above)
Comment (guimachiavelli) [reply learn/async/paginating_tasks.mdx:23]: `total` indicates the total number of tasks matching the query. Meaning, if my instance has 50 tasks and I request the first 20, `total` will be `50`. I'll update the text with that info.

File: .code-samples.meilisearch.yaml (hunk 3 above)
Comment (curquiza) [reply .code-samples.meilisearch.yaml:969]: Oh yes sorry I read wrongly!

**Pull request merged**